src/webview_provider/newLineagePanel.ts

```diff
@@ -759,6 +789,7 @@ export class NewLineagePanel implements LineagePanelView {
         upstreamCount,
         downstreamCount,
         nodeType,
+        isExternalProject: this.isExternalProject(_node, projectName),
```

Comment:
This is for sources. I beleive we could refer sources from other projects as well, if not needed @mdesmet pls lmk

Reply:
In dbt cloud this is not allowed but I think in dbt loom it is supported as those sources become part of the manifest.